Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:13.12-alpine
+
+WORKDIR /usr/src/app
+
+COPY ["package.json", "package-lock.json", "./"]
+RUN npm install --silent
+COPY . .
+
+RUN npm run build
+CMD npm run serve

```

Comment:
create a multistage `Dockerfile`, so running the image will only run the compiled js files and not compile the ts.